larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -653,29 +653,7 @@ StatusCode MasterAlgorithm::Copy(const Pandora *const pPandora, const CaloHit *c
         return STATUS_CODE_INVALID_PARAMETER;
     }
     LArCaloHitParameters parameters;
-    parameters.m_positionVector = pCaloHit->GetPositionVector();
-    parameters.m_expectedDirection = pCaloHit->GetExpectedDirection();
-    parameters.m_cellNormalVector = pCaloHit->GetCellNormalVector();
-    parameters.m_cellGeometry = pCaloHit->GetCellGeometry();
-    parameters.m_cellSize0 = pCaloHit->GetCellSize0();
-    parameters.m_cellSize1 = pCaloHit->GetCellSize1();
-    parameters.m_cellThickness = pCaloHit->GetCellThickness();
-    parameters.m_nCellRadiationLengths = pCaloHit->GetNCellRadiationLengths();
-    parameters.m_nCellInteractionLengths = pCaloHit->GetNCellInteractionLengths();
-    parameters.m_time = pCaloHit->GetTime();
-    parameters.m_inputEnergy = pCaloHit->GetInputEnergy();
-    parameters.m_mipEquivalentEnergy = pCaloHit->GetMipEquivalentEnergy();
-    parameters.m_electromagneticEnergy = pCaloHit->GetElectromagneticEnergy();
-    parameters.m_hadronicEnergy = pCaloHit->GetHadronicEnergy();
-    parameters.m_isDigital = pCaloHit->IsDigital();
-    parameters.m_hitType = pCaloHit->GetHitType();
-    parameters.m_hitRegion = pCaloHit->GetHitRegion();
-    parameters.m_layer = pCaloHit->GetLayer();
-    parameters.m_isInOuterSamplingLayer = pCaloHit->IsInOuterSamplingLayer();
-    // ATTN Parent of calo hit in worker is corresponding calo hit in master
-    parameters.m_pParentAddress = static_cast<const void *>(pCaloHit);
-    parameters.m_larTPCVolumeId = pLArCaloHit->GetLArTPCVolumeId();
-    parameters.m_daughterVolumeId = (m_larCaloHitVersion > 1) ? pLArCaloHit->GetDaughterVolumeId() : 0;
+    pLArCaloHit->FillParameters(parameters);
 
```

Comment:
Could lose empty line here, for consistency with LArMCParticle style, below. (Barely worth mentioning!)